schema/me/conversation.js

```diff
@@ -74,6 +77,28 @@ export const ConversationParticipantType = new GraphQLObjectType({
   },
 })
 
+export const ConversationResponderType = new GraphQLObjectType({
```

Comment:
And I think the behavior could change even more in the future (i.e. the differences in behavior between a Partner responding to inquiries and a User initiating them).